runtime/native/class.cpp

```diff
@@ -12,17 +13,22 @@
 #include "rep.h"
 #include "monitor.h"
 #include "threads.h"
+
 #include <algorithm>
 #include <cassert>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
 #include <dlfcn.h>
-#include <gc.h>
 #include <stdexcept>
 #include <stdio.h>
 #include <string>
 #include <unordered_map>
+#include <pthread.h>
+
+#define GC_THREADS
```

Comment:
I'd include a comment to explain what you're doing with the weird GC_THREADS inclusion behavior, so that we remember why its necessary :)

OR this should go in the developer guide /github issue or something since it shows up in a bunch of files.

Reply:
I documented this in the developer guide (b7b1bc0). I will add more about concurrency to the developer guide in the later PR.